<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="windowOpacity">
   <double>0.980000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #3A4655;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>0</y>
      <width>350</width>
      <height>80</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
qproperty-aligment: 'AlingVCenter | AlignRight';
  font-size: 30px;
color: #fff;
}
  

</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_C">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 25%;
  background: #425062;
  color: #fff;
  padding: 20px;
  display: inline-block;
  font-size: 25px;
  text-align: center;
  vertical-align: middle;
  margin-right: -4px;
  border-right: solid 2px #3C4857;
  border-bottom: solid 2p x #3C4857;color: #D95D4E;
  background: #404D5E;</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_PM">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 25%;
  background: #425062;
  color: #fff;
  padding: 20px;
  display: inline-block;
  font-size: 25px;
  text-align: center;
  vertical-align: middle;
  margin-right: -4px;
  border-right: solid 2px #3C4857;
  border-bottom: solid 2p x #3C4857;color: #AEB3BA;
  background: #404D5E;</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_PC">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 25%;
  background: #425062;
  color: #fff;
  padding: 20px;
  display: inline-block;
  font-size: 25px;
  text-align: center;
  vertical-align: middle;
  margin-right: -4px;
  border-right: solid 2px #3C4857;
  border-bottom: solid 2p x #3C4857;color: #AEB3BA;
  background: #404D5E;</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DIV">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 25%;
  background: #425062;
  color: #fff;
  padding: 20px;
  display: inline-block;
  font-size: 25px;
  text-align: center;
  vertical-align: middle;
  margin-right: -4px;
  border-right: solid 2px #3C4857;
  border-bottom: solid 2p x #3C4857;color: #AEB3BA;
  background: #404D5E;</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 25%;
  background: #425062;
  color: #fff;
  padding: 20px;
  display: inline-block;
  font-size: 25px;
  text-align: center;
  vertical-align: middle;
  margin-right: -4px;
  border-right: solid 2px #3C4857;
  border-bottom: solid 2px #3C4857;</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 25%;
  background: #425062;
  color: #fff;
  padding: 20px;
  display: inline-block;
  font-size: 25px;
  text-align: center;
  vertical-align: middle;
  margin-right: -4px;
  border-right: solid 2px #3C4857;
  border-bottom: solid 2px #3C4857;</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_MULT">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 25%;
  background: #425062;
  color: #fff;
  padding: 20px;
  display: inline-block;
  font-size: 25px;
  text-align: center;
  vertical-align: middle;
  margin-right: -4px;
  border-right: solid 2px #3C4857;
  border-bottom: solid 2p x #3C4857;color: #AEB3BA;
  background: #404D5E;</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 25%;
  background: #425062;
  color: #fff;
  padding: 20px;
  display: inline-block;
  font-size: 25px;
  text-align: center;
  vertical-align: middle;
  margin-right: -4px;
  border-right: solid 2px #3C4857;
  border-bottom: solid 2px #3C4857;</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_MIN">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>260</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 25%;
  background: #425062;
  color: #fff;
  padding: 20px;
  display: inline-block;
  font-size: 25px;
  text-align: center;
  vertical-align: middle;
  margin-right: -4px;
  border-right: solid 2px #3C4857;
  border-bottom: solid 2p x #3C4857;color: #AEB3BA;
  background: #404D5E;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 25%;
  background: #425062;
  color: #fff;
  padding: 20px;
  display: inline-block;
  font-size: 25px;
  text-align: center;
  vertical-align: middle;
  margin-right: -4px;
  border-right: solid 2px #3C4857;
  border-bottom: solid 2px #3C4857;</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 25%;
  background: #425062;
  color: #fff;
  padding: 20px;
  display: inline-block;
  font-size: 25px;
  text-align: center;
  vertical-align: middle;
  margin-right: -4px;
  border-right: solid 2px #3C4857;
  border-bottom: solid 2px #3C4857;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 25%;
  background: #425062;
  color: #fff;
  padding: 20px;
  display: inline-block;
  font-size: 25px;
  text-align: center;
  vertical-align: middle;
  margin-right: -4px;
  border-right: solid 2px #3C4857;
  border-bottom: solid 2px #3C4857;</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_PL">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>350</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 25%;
  background: #425062;
  color: #fff;
  padding: 20px;
  display: inline-block;
  font-size: 25px;
  text-align: center;
  vertical-align: middle;
  margin-right: -4px;
  border-right: solid 2px #3C4857;
  border-bottom: solid 2p x #3C4857;color: #AEB3BA;
  background: #404D5E;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 25%;
  background: #425062;
  color: #fff;
  padding: 20px;
  display: inline-block;
  font-size: 25px;
  text-align: center;
  vertical-align: middle;
  margin-right: -4px;
  border-right: solid 2px #3C4857;
  border-bottom: solid 2px #3C4857;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 25%;
  background: #425062;
  color: #fff;
  padding: 20px;
  display: inline-block;
  font-size: 25px;
  text-align: center;
  vertical-align: middle;
  margin-right: -4px;
  border-right: solid 2px #3C4857;
  border-bottom: solid 2px #3C4857;</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>350</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 25%;
  background: #425062;
  color: #fff;
  padding: 20px;
  display: inline-block;
  font-size: 25px;
  text-align: center;
  vertical-align: middle;
  margin-right: -4px;
  border-right: solid 2px #3C4857;
  border-bottom: solid 2px #3C4857;</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_EQ">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>440</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 25%;
  background: #425062;
  color: #fff;
  padding: 20px;
  display: inline-block;
  font-size: 25px;
  text-align: center;
  vertical-align: middle;
  margin-right: -4px;
  border-right: solid 2px #3C4857;
  border-bottom: solid 2p x #3C4857;color: #AEB3BA;
  background: #404D5E;</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_D">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>91</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 25%;
  background: #425062;
  color: #fff;
  padding: 20px;
  display: inline-block;
  font-size: 25px;
  text-align: center;
  vertical-align: middle;
  margin-right: -4px;
  border-right: solid 2px #3C4857;
  border-bottom: solid 2p x #3C4857;color: #AEB3BA;
  background: #404D5E;</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_B">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>440</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 25%;
  background: #425062;
  color: #fff;
  padding: 20px;
  display: inline-block;
  font-size: 25px;
  text-align: center;
  vertical-align: middle;
  margin-right: -4px;
  border-right: solid 2px #3C4857;
  border-bottom: solid 2p x #3C4857;color: #AEB3BA;
  background: #404D5E;</string>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>440</y>
      <width>91</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 25%;
  background: #425062;
  color: #fff;
  padding: 20px;
  display: inline-block;
  font-size: 25px;
  text-align: center;
  vertical-align: middle;
  margin-right: -4px;
  border-right: solid 2px #3C4857;
  border-bottom: solid 2px #3C4857;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
